packages/edit-site/src/components/add-new-template/new-template.js

```diff
@@ -52,13 +86,18 @@ export default function NewTemplate( { postType } ) {
 				slug,
 			} );
 
+			const newTemplateContent =
+				defaultBlockTemplate ??
+				getFallbackTemplateContentFromHierarchy( slug, templates );
```

Comment:
It's a bit weird to rely on existing templates to create a fallback template content? Is this how it's supposed to work or should we actually provide a list of default fallbacks?

Reply:
The motivation for using existing templates is to ensure that the new template matches the current theme design (IE include the appropriate header / footer template parts and general block structure).

A much simpler approach would be to prescribe a template including the base header / footer template parts, then the user would need to select which one to use whilst creating the template. So a new search template might look like:

<img width="1638" alt="Screenshot 2021-12-08 at 10 06 45" src="https://user-images.githubusercontent.com/846565/145189551-bec503c2-decb-4b36-808e-1f5e2f0f05e3.png">

The drawback here is that if the theme has a complex layout (maybe the header is actually a sidebar) then this won't match at all.